{"type": "end", "timestamp": "2024-12-13T18:18:40.877766", "step_id": "command-1317193477083369564.main.BotCommands._send_report", "task_id": "command-1317193477083369564.main", "result": null, "exception": {"type": "builtins.AssertionError", "args": ["start != end"], "details": "Traceback (most recent call last):\n  File \"C:\\Users\\rache\\miniconda3\\envs\\rubber-duck\\Lib\\site-packages\\quest\\historian.py\", line 565, in handle_step\n    result = await result\n             ^^^^^^^^^^^^\n  File \"C:\\Users\\rache\\Documents\\Bryce\\rubber-duck\\rubber_duck\\discord_bot.py\", line 254, in send_message\n    if view is not None:\n       ^^^^\n  File \"_pydevd_bundle\\\\pydevd_pep_669_tracing_cython.pyx\", line 539, in _pydevd_bundle.pydevd_pep_669_tracing_cython.py_line_callback\n  File \"_pydevd_bundle\\\\pydevd_pep_669_tracing_cython.pyx\", line 535, in _pydevd_bundle.pydevd_pep_669_tracing_cython.py_line_callback\n  File \"C:\\Program Files\\JetBrains\\PyCharm 2024.3\\plugins\\python-ce\\helpers\\pydev\\pydevd.py\", line 1220, in do_wait_suspend\n    self._do_wait_suspend(thread, frame, event, arg, suspend_type, from_this_thread)\n  File \"C:\\Program Files\\JetBrains\\PyCharm 2024.3\\plugins\\python-ce\\helpers\\pydev\\pydevd.py\", line 1235, in _do_wait_suspend\n    time.sleep(0.01)\nKeyboardInterrupt\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"C:\\Users\\rache\\miniconda3\\envs\\rubber-duck\\Lib\\site-packages\\quest\\historian.py\", line 565, in handle_step\n    result = await result\n             ^^^^^^^^^^^^\n  File \"C:\\Users\\rache\\Documents\\Bryce\\rubber-duck\\rubber_duck\\bot_commands.py\", line 117, in _send_report\n    await self._send_message(channel_id, f'Graph: {report_string}', file=img)\n  File \"C:\\Users\\rache\\miniconda3\\envs\\rubber-duck\\Lib\\site-packages\\quest\\wrappers.py\", line 18, in new_func\n    return await find_historian().handle_step(func.__name__, func, *args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\rache\\miniconda3\\envs\\rubber-duck\\Lib\\site-packages\\quest\\historian.py\", line 617, in handle_step\n    _prune(step_id, self._history)\n  File \"C:\\Users\\rache\\miniconda3\\envs\\rubber-duck\\Lib\\site-packages\\quest\\historian.py\", line 129, in _prune\n    assert record['type'] == 'end', f'{record[\"type\"]} != end'\n           ^^^^^^^^^^^^^^^^^^^^^^^\nAssertionError: start != end\n"}}